src/plugins/memory/memory.yml

```diff
@@ -10,3 +10,12 @@ tileA:
 tileB:
   en: Card B
   de: Karte B
+text:
+  en: text
+  de: Text
+url:
+  en: media URL
+  de: Medien-URL
+preview:
+  en: Card pairs preview
+  de: Kartenpaarevorschau
```

Comment:
The preview just shows one pair at a time, so we might wanna go with "Card pair preview" / "Kartenpaarvorschau".

Reply:
True, makes sense, corrected.